### ISSUE TITLE
Alternative Pool to public-pool?

### ISSUE THREAD

**Issue by dapdahang** [BitMaker-hub/NerdMiner_v2#316]
With public-pool down now for more than 24 hours, are there good alternatives where I can also check my hashrate online? Ckpool apparently is not able to show the live stats.
Thanks guys

**Comment by 19201003080114** [comment]
stratum+tcp://pool.vkbit.com:3333

**Comment by sany3001** [comment]
Have you predicted future @dapdahang? For me went https://web.public-pool.io/ down just some minutes ago local CET time.

**Comment by dapdahang** [comment]
Yeah, while it was running for me today again, it went down again like 20
minutes ago.
I am now trying vkbit, so far looking good



Am Do., 8. Feb. 2024 um 08:46 Uhr schrieb sany3001 ***@***.***

**Comment by jorgb90** [comment]
for me its not saving pool address when in config mode. It always goes back to airdns..

**Comment by KaSt** [comment]
I just opened mine, after public-pool's semi downtime days ago and I also wanted to have mine. Using  public-pool's source code. In case the Developer is interested and agrees in me posting it here, I shall do it. Still in test, checking stability and at the moment only my miners (4) are on it.
3 NerdMiners running on:
- 2 LilyGo Display 1.4
- 1 LilyGo QT-Pro

Also have:
- 1 PIZero with four old USB miners, for a total of around 3Gh/s.

Good old days when I could mine with the CPU of my computer.

**Comment by mynameismichele** [comment]
I also tried running my own [public-pool](https://github.com/benjamin-wilson/public-pool) on umbrel and it is working fine so far.
It's a pity that the total hashrate showed on ESP32-2432S028R is not working anymore because I think it was connected only to public-pool.io , any chance we can get it back @nitroxgas?
